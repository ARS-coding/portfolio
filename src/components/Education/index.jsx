import React from 'react';

const Education = () => (
  <>
    <h2 id="education">Education</h2>
    <p>
      In 2020, Tarkan MansuroÄŸlu graduated from{' '}
      <a target="_blank" rel="noreferrer" href="https://www.re-coded.com/">
        Re:Coded
      </a>
      &apos;s front end web development bootcamp, which is a collaboration
      between{' '}
      <a target="_blank" rel="noreferrer" href="https://flatironschool.com/">
        Flatiron School
      </a>{' '}
      and Re:Coded. Prior to his coding education, he was a finance graduate. In
      2019, he graduated from international finance program of{' '}
      <a target="_blank" rel="noreferrer" href="https://www.emu.edu.tr/en">
        Eastern Mediterranean University(EMU)
      </a>{' '}
      as top student with 4.00 out of 4.00 CGPA.
    </p>
  </>
);

export default Education;
