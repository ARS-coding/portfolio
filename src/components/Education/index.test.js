import React from 'react';
import renderer from 'react-test-renderer';
import Education from '.';

it('Education renders correctly', () => {
  const tree = renderer.create(<Education />).toJSON();
  expect(tree).toMatchSnapshot();
});
