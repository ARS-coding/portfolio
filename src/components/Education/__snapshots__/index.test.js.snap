// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education renders correctly 1`] = `
Array [
  <h2
    
    id="education"
  >
    Education
  </h2>,
  <p>
    In 2020, Tarkan MansuroÄŸlu graduated from
     
    <a
      href="https://www.re-coded.com/"
      rel="noreferrer"
      target="_blank"
    >
      Re:Coded
    </a>
    's front end web development bootcamp, which is a collaboration between
     
    <a
      href="https://flatironschool.com/"
      rel="noreferrer"
      target="_blank"
    >
      Flatiron School
    </a>
     
    and Re:Coded. Prior to his coding education, he was a finance graduate. In 2019, he graduated from international finance program of
     
    <a
      href="https://www.emu.edu.tr/en"
      rel="noreferrer"
      target="_blank"
    >
      Eastern Mediterranean University(EMU)
    </a>
     
    as top student with 4.00 out of 4.00 CGPA.
  </p>,
]
`;
